sample/app-kotlin/build.gradle

```diff
@@ -0,0 +1,43 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-kapt'
+apply plugin: 'com.fourlastor.pickle'
+
+android {
+    compileSdkVersion 28
+    defaultConfig {
+        applicationId "com.fourlastor.pickle.sample"
+        minSdkVersion 21
+        targetSdkVersion 28
+        versionCode 1
+        versionName "1.0"
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+    }
+
+    sourceSets {
+        androidTest.java.srcDir 'build/generated/source/pickle/androidTest/debug'
```

Comment:
So good news and bad news 🎉 

I was able to solve the Kotlin issue with this change in the sample itself. 

I also tried bunch of things including doing this automatically in the plugin (eventhough it would be hacky) but it didn't even work. The problem is with `registerJavaGeneratingTask` in the Plugin code. Somehow this is not picked up by Kotlin compiler.